docs/themes/psh-docs/layouts/shortcodes/endpoint-description.md

```diff
@@ -19,20 +19,22 @@
 To define the service, use {{ if eq ($type) "mariadb" }}
   the `{{ $type }}` or `mysql` type for MariaDB or the `oracle-mysql` type for Oracle MySQL
   {{ else if eq $type "redis" }}
-  the `{{ $type }}` type for ephemeral Redis
+  the `{{ $type }}` type for persistent Redis
   {{ else }}
   the `{{ $type }}` type{{ end }}:
 
 <!-- Create an example services.yaml file from data in the registry. -->
 {{ partial "examples/servicedefn" $data }}
 
-{{ if eq $type "redis" }}
-Alternatively, use the `redis-persistent` type for persistent Redis:
+{{ if eq $type "redis-persistent" }}
+Note that persistent Redis requires a disk to store data.
```

Comment:
```suggestion
Note that persistent Redis requires `disk` to store data.
```

Reply:
Also, since I can't comment on what you haven't changed, I think we need to update [this line](https://github.com/platformsh/platformsh-docs/pull/3318/files#diff-4304e44d631e4b1504a05dc250212f7de83833c75ac42ca8e149988f1ef573feL71) in this file as well.

Called with

```
{{% endpoint-description type="redis-persistent" noApp=true onlyLanguage="php" php=true /%}}
```

`type` is used for service type `redis-persistent` (which is correct) AND the PHP extension name `redis-persistent` (which is incorrect, it should still just be `redis`)

![CleanShot 2023-08-03 at 08 38 03](https://github.com/platformsh/platformsh-docs/assets/5473659/394548a0-8fd2-4be9-a814-dc1973f19b3d)
![CleanShot 2023-08-03 at 08 38 18](https://github.com/platformsh/platformsh-docs/assets/5473659/e5b95dd7-0fe8-4de5-9b52-1ab5d78899a0)


I think we can add another catch around line 80

```
  {{ if eq $type "redis-persistent" }}
   {{ $extension_name = "redis" }}
  {{ end }}
```